newsletter_automation/helpers/authentication_required.py

```diff
@@ -3,7 +3,11 @@
 """
 
 from functools import wraps
-from flask import render_template, session
+from flask import render_template, session, request, jsonify
+import conf.apikey_conf as conf
```

Comment:
The variable name `conf` is too general. Further, `conf` is also the name of a directory. Import it at as something else. Your code should be readable. Right now, lines like `API = conf.API_KEY` does not make sense especially if we know `conf` is a directory name.